tests/paillier.rs

```diff
@@ -140,11 +140,11 @@ fn bytes() {
 
     let bytes = pk.to_bytes();
     assert_eq!(bytes.len(), 256);
-    let pk1 = EncryptionKey::from_bytes(bytes.as_slice());
+    let pk1 = EncryptionKey::from_bytes(bytes.as_slice()).unwrap();
```

Comment:
How the hell did this test compile before you added `unwrap`?? 😛

Reply:
It didn't...
There is no CI setup on the original repo, so it probably got through in some commit.

Reply:
yikes 😆